core/src/SYCL/Kokkos_SYCL_Instance.hpp

```diff
@@ -75,6 +75,12 @@ class SYCLInternal {
 
   std::optional<sycl::queue> m_queue;
 
+  // Using std::vector<std::optional<sycl::queue>> reveals a compiler bug when
+  // compiling for the CUDA backend. Storing pointers instead works around this.
```

Comment:
Where did we report this bug?  @nliber

Reply:
I don't know that we have (where do SYCL+CUDA bugs get reported)?